### PR TITLE
fix margin for close in participants; fix auto close for Kb.Dropdown

### PR DESCRIPTION
**Pull request by songgao**
I didn't get to test on mobile, as I don't think the `Dropdown` is used anywhere on mobile other than in the wallet, which doesn't seem ready yet.

### CODE REVIEW

**Comment by songgao** [comment]
bump!

File: shared/common-adapters/dropdown.js [hunk 1]
```diff
@@ -1,39 +1,35 @@
 // @flow
 import * as React from 'react'
 import Box from './box'
-import PopupDialog from './popup-dialog'
+import Overlay from './overlay'
+import OverlayParentHOC, {type OverlayParentProps} from './overlay/parent-hoc'
+
```
Comment (buoyad) [inline shared/common-adapters/dropdown.js]: 🔥

File: shared/common-adapters/dropdown.js [hunk 2]
```diff
@@ -44,88 +40,97 @@ class Dropdown extends React.Component<Props, State> {
...
-          )}
-          <Box style={{...globalStyles.flexBoxCenter, minHeight: isMobile ? 48 : 32, width: '100%'}}>
+      <Box style={Styles.collapseStyles([{width: Styles.isMobile ? '100%' : 270}, this.props.style])}>
+        <ButtonBox
+          onClick={!this.props.disabled ? this._toggleOpen : null}
+          disabled={this.props.disabled}
+          ref={this.props.setAttachmentRef}
+        >
+          <Box
+            style={{
```
Comment (buoyad) [inline shared/common-adapters/dropdown.js]: Could this go in `styles`?

**Comment by songgao** [comment]
@buoyad thanks! Would you mind taking another look? I added `center center` as well as a bunch of `Dropdown` stories.

**Pull request merged**